build.gradle

```diff
@@ -19,5 +20,7 @@ allprojects {
     repositories {
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```

Comment:
Why do we need JitPack here?

Reply:
oops, sorry about this,I have fork it a long time ago and now create PR. I'll create another PR to fix this issue.